### ISSUE TITLE
No aarch64 build available

### ISSUE THREAD

**Issue by akallabeth** [flathub/im.riot.Riot#167]
Would be nice to have this on arm too

**Comment by SISheogorath** [comment]
The reason for not being available on aarch64 is that we currently just re-package the official `.deb` files from upstream. If these are available as aarch64 it should be easy to built it for this architecture as well.

Otherwise we need to revisit the build process entirely, which I already did, but stayed unsuccessful for now. Electron packaging offline is a beast.

**Comment by t3chguy** [comment]
> The reason for not being available on aarch64 is that we currently just re-package the official .deb files from upstream. If these are available as aarch64 it should be easy to built it for this architecture as well.

This is no longer the case AIUI, the tarballs are used instead. Tarballs for aarch64 were unavailable for some time due to it being seemingly impossible to crosscompile seshat from x64 to aarch64 in bundled mode (dynamic was fine for the debs)

https://github.com/element-hq/element-desktop/pull/1664 should be around the corner now to bring the missing aarch64 tarball to unblock this.
